README.md

```diff
@@ -1,9 +1,20 @@
-# WIP Rust BLE stack
+# rubble
+
+> A WIP Rust BLE stack
 
 [![Build Status](https://travis-ci.org/jonas-schievink/rubble.svg?branch=master)](https://travis-ci.org/jonas-schievink/rubble)
+[![Line Count](https://tokei.rs/b1/github/jonas-schievink/rubble)](https://github.com/XAMPPRocky/tokei)
```

Comment:
I'm not a big fan of bragging about line counts. They don't really convey much information, and might even keep away new contributors when the numbers look too large.

Reply:
Good point, removed.